data_dumps_dstb_episodes.csv

```diff
@@ -0,0 +1 @@
Other ICT System,self,adherence_type_ict_other_detail
Treatment Supporter Designation,self,treatment_supporter_designation
Treatment Supporter First Name,self,treatment_supporter_first_name
Treatment Supporter Last Name,self,treatment_supporter_last_name
Treatment Supporter Mobile Number,self,treatment_supporter_mobile_number
Weight,self,weight
Height,self,height
Dosage Frequency,self,adherence_schedule_id
Drug Formulations,self,drug_formulations
Loose Drugs Dosage,self,dosage_loose_drugs
Weight Band,self,weight_band
"Drug Packaging, if any",self,drug_packaging
"FDC Dosage per Day, if any",self,dosage_fdc
HIV status,N/A,N/A
"Date of HIV Test, if Reactive & Non - Reactive",N/A,N/A
"PID Number, if any",N/A,N/A
CPT Initiated?,N/A,N/A
"Date of CPT Initiation?, if yes",N/A,N/A
ART Initiated?,N/A,N/A
Date of ART Initiation?,N/A,N/A
"ART Number, if any",N/A,N/A
"ART Centre Name, if any",N/A,N/A
"HIV Testing offered?, if HIV status is Unknown",N/A,N/A
Diabetic Status,N/A,N/A
RBS Level,N/A,N/A
FBS Level,N/A,N/A
Initiated on Anti-diabetes treatment?,N/A,N/A
Date of anti-diabetes treatment initiation,N/A,N/A
Anti diabetes treatment program number,N/A,N/A
"Diabetes testing offered?, if Diabetes Status is Unknown",N/A,N/A
Other Comorbidity,N/A,N/A
"Specify Other Co-morbidity, if selected as Yes",N/A,N/A
Current Tobacco user?,N/A,N/A
Tobacco Type?,N/A,N/A
Linked to cessation program,N/A,N/A
History of Alcohol intake,N/A,N/A
Linked for deaddiction Program?,N/A,N/A
Has Nikshay ID?,N/A,N/A
Enter Nikshay ID,N/A,N/A
Has TB Notification Number for current episode,N/A,N/A
Enter TB Number,self,tb_number
Marital Status,N/A,N/A
Marital Status (Other - Specify),N/A,N/A
Socio Economic Status,N/A,N/A
Name & Address of Contact Person,N/A,N/A
Mobile/ Landline Number of Contact Person,N/A,N/A
Predominant Symptoms,self,predominant_symptom
"Duration of symptoms in Days, if any",self,symptom_duration
History of ATT greater than one month?,N/A,N/A
Number of health care providers visited before diagnosis for current episodes,N/A,N/A
"Months of Treatment undertaken
",N/A,N/A
Months since treatment for last episode,N/A,N/A
Source of treatment for previous regimen,N/A,N/A
Key population,N/A,N/A
Occupation,N/A,N/A
Current Address - State,N/A,N/A
Current Address - District,N/A,N/A
Current Address - Block/ Taluka/ Mandal,N/A,N/A
Current Address - Village/ Town/ City,N/A,N/A
Current Address - Address First Line,N/A,N/A
Current Address - Postal code,N/A,N/A
Current Address - Landmark,N/A,N/A
Initiate Continuation Phase?,self,cp_initiated
"Date of CP Initiation
",self,cp_initiation_date
Adverse drug reaction history,self,adr_history
"Date of first reaction/ symptom, if any",N/A,N/A
"Specify reaction, if any",N/A,N/A
"Note on Reactions, if any",N/A,N/A
Any Action taken,N/A,N/A
Date of remedial action,N/A,N/A
"Specify action, if any",N/A,N/A
Duration of management for adverse events (Days),N/A,N/A
Treatment Outcome,self,treatment_outcome
Date of Treatment Outcome,self,treatment_outcome_date
```

Comment:
Was this file committed by mistake?  It's in the root of the repo, and there's a duplicate in the mgmt cmd dir

Reply:
eh? where did that come from.. thanks for catching.. will remove it.